src/sheet-root.tsx

```diff
@@ -105,7 +105,8 @@ const showRadioListNativeSheet = ({
     (nativeSheetImplementation as NativeSheetImplementation)({
         title,
         subtitle,
-        description,
+        // TODO: add multiline support to native sheet
+        description: Array.isArray(description) ? description.join('\n') : description,
```

Comment:
perhaps, you should join with two line breaks, to simulate different paragraphs

Reply:
also note that:

`[].join('\n')` returns empty string, but in this case we probably want to set an `undefined` description.

Reply:
I want to do some tests with the native implementation and see what happens with `\n` and `\n\n`

Reply:
Tested. It seems to work quite fine with the double `\n\n`:
![image](https://github.com/Telefonica/mistica-web/assets/1849576/9204d2ff-fc17-41ae-a466-4aa7b0a178d8)